plugins/GSdx/GSDeviceOGL.cpp

```diff
@@ -241,7 +241,7 @@ GSTexture* GSDeviceOGL::CreateSurface(int type, int w, int h, bool msaa, int fmt
 	GL_PUSH("Create surface");
 
 	// A wrapper to call GSTextureOGL, with the different kind of parameter
-	GSTextureOGL* t = new GSTextureOGL(type, w, h, fmt, m_fbo_read, m_mipmap > 1 || m_filter != TriFiltering::None);
+	GSTextureOGL* t = new GSTextureOGL(type, w, h, fmt, m_fbo_read, (m_mipmap > 1 || auto_mipmap_full) || m_filter != TriFiltering::None);
```

Comment:
create an intermediate variable for `(m_mipmap > 1 || auto_mipmap_full) || m_filter != TriFiltering::None`. And you don't need the parenthesis